packages/fxa-auth-server/lib/senders/templates/index.js

```diff
@@ -47,6 +49,13 @@ async function init(log) {
     return reduceOp(arguments, (a, b) => a || b);
   }
 
+  function isEqualHelper(a, b, options) {
+    if (a === b) {
+      return options.fn(this);
+    }
+    return options.inverse(this);
+  }
```

Comment:
Consistency suggestion: using the pattern from `orHelper` above we can write
```
function isEqualHelper() {
  return reduceOp(arguments, (a, b) => a === b);
}
```

Reply:
I can't find an instance of `orHelper` actually being used? I did try to follow that pattern initially but it results in this: 
<img width="673" alt="Screen Shot 2021-03-01 at 7 51 02 PM" src="https://user-images.githubusercontent.com/22355127/109584595-7716de00-7ac7-11eb-8110-8cc5445df40c.png">

Reply:
🤦 Sorry about that.  In that case, please delete that `orHelper` dead code.

Reply:
ok so - I was mistaken! This helper _is_ actually used [here](https://github.com/mozilla/fxa/blob/5c22cafd576ff7bcc28e98cb801297dde9579ca6/packages/fxa-auth-server/lib/senders/templates/downloadSubscription.html#L90).  I had been looking for `{{#or}}` but the usage was just a bit different. I've updated ifEqualHelper to match.